<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-0PPIB21" ignored="0" name="Test" tests="2" failures="2" timestamp="2021-04-17T16:43:15 IST" time="15.982" errors="0">
  <testcase name="invalidLogin" time="3.853" classname="com.web.automationpractice.loginTest">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at com.web.automationpractice.loginTest.invalidLogin(loginTest.java:55)
at java.util.ArrayList.forEach(ArrayList.java:1257)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- invalidLogin -->
  <testcase name="invalidLogin" time="9.723" classname="com.web.automationpractice.loginTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at testBase.tb.getScreenShot(tb.java:40)
at report.Testlistner.onTestFailure(Testlistner.java:19)
at java.util.ArrayList.forEach(ArrayList.java:1257)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- invalidLogin -->
</testsuite> <!-- Test -->
